spec/serializers/admin/subscription_customer_serializer_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe Api::Admin::SubscriptionCustomerSerializer do
+  let(:distributor) { create(:distributor_enterprise) }
+  let(:delivery) { create(:shipping_method, require_ship_address: true, distributors: [distributor]) }
+  let(:email) { 'some@email.com' }
+  let(:address) { create(:address) }
+  let(:customer) { create(:customer, email: email) }
+  let(:order) { create(:completed_order_with_totals, email: email, user: nil, customer: nil, distributor: distributor) }
```

Comment:
do we actually need all these models to be persisted in DB for the serializer to work? can we use FactoryBot's `build_stubbed` method instead?

 If we avoid hitting DB we would save precious time (and our test suite is way too slow already) in this single test as the number of objects being used here is considerable.

Reply:
Yeah, you're right. I am testing stuff that is already covered by the `AddressFinder` spec, I've stubbed out the address finder in the spec, 2.5sec > 0.8 sec.

Reply:
Done